src/pages/organize/[orgId]/projects/[campId]/events/[eventId]/participants.tsx

```diff
@@ -42,18 +43,27 @@ const ParticipantsPage: PageWithLayout<ParticipantsProps> = ({
   const dataModel = useModel(
     (env) => new EventDataModel(env, parseInt(orgId), parseInt(eventId))
   );
+
   return (
     <ZUIFuture future={dataModel.getData()}>
-      {() => {
+      {(data) => {
         return (
-          <Grid container spacing={2}>
-            <Grid item md={8} xs={12}>
-              <ParticipantSummaryCard model={dataModel} />
+          <>
+            <Grid container spacing={2}>
```

Comment:
The `<AddPersonButton>` should be at the top of the screen.

## This PR now
![image](https://user-images.githubusercontent.com/550212/235070944-e952e13e-6f38-4704-aeac-39a0305fe3a8.png)

## Design mockups
![image](https://user-images.githubusercontent.com/550212/235071171-50b2c0c9-1738-43a2-820c-a819a0d0b7ea.png)